Client/Configuration/Base.xcconfig

```diff
@@ -10,7 +10,7 @@
 #include "Local/BuildId.xcconfig"
 #include "Local/Version.xcconfig"
 
-OTHER_SWIFT_FLAGS_BASE=-DMOZ_TARGET_$(TARGET_NAME:upper) -DNO_USER_WALLETS -DNO_SKUS
```

Comment:
Let's keep this flag uncommitted until the feature is fully complete

Reply:
Done!